client/src/app/site/pages/meetings/services/media-manage.service.ts

```diff
@@ -68,10 +56,15 @@ export class MediaManageService {
     }
 
     public getLogoUrl(place: LogoPlace): string | null {
-        // Note: we are not fetching the mediafile view model at any place.
-        const mediafileId = this.activeMeetingService.meeting?.logo_id(place);
+        // Note: we are not fetching the mediafile view model at any place except when filtering for the defaults.
```

Comment:
Update the comment, the defaults no longer exist

Reply:
No, logos can still have a default

Reply:
Ah, the web_header is the default, right